data/ssystem_major.ini

```diff
@@ -2002,8 +2002,8 @@ tex_map=obs_saturn.jpg
 ; orbit_Inclination = 90
 ; orbit_MeanLongitude = 90
 ; orbit_Period = 70000000000
-; set to 9.6AU: Average light time as seen from earth.
-orbit_SemiMajorAxis = 1436160000
+; 30 Mio km to see Phoebe's orbit
```

Comment:
Mio km seems to be a rarely used designation for the standard Gm. At least, I didn't find much use of it online, and most of occurrences were in the context of German language.

Reply:
Although few humans understand Gm. I will use M km in these comments.

Reply:
Well, just over half a century ago few humans understood nm, using mμ instead.